ablateLibrary/mathFunctions/parsedSeries.cpp

```diff
@@ -122,7 +122,7 @@ PetscErrorCode ablate::mathFunctions::ParsedSeries::ParsedPetscSeries(PetscInt d
         parser->coordinate[1] = 0;
         parser->coordinate[2] = 0;
 
-        for (auto d = 0; d < std::min(dim, 3); d++) {
+        for (auto d = 0; d < PetscMin(dim, 3); d++) {
```

Comment:
Suggest changing to
```suggestion
        for (PetscInt d = 0; d < PetscMin(dim, 3); d++) {
```
Similar change on line 130 of ablateLibrary/mathFunctions/parsedNested.cpp